cioos_data_transform/cioos_data_transform/OceanNcVar.py

```diff
@@ -483,6 +485,12 @@ def add_var(self, varlist):
             # self.standard_name = ''
             self.long_name = "Sigma-theta"
             self.units = bodc_units
+        elif self.type in ['DOUB', 'SING', 'SYTM', 'INTE']:
+            type_mapping = {'DOUB': 'float64',
```

Comment:
looks like this mapping is defined twice (see `odf_dtypes()`) , maybe it can be reused?